.eslintrc.cjs

```diff
@@ -19,7 +19,15 @@ const config = {
       "./www/tsconfig.json",
     ],
   },
+  overrides: [
+    // Template files don't have reliable type information
+    {
+      files: ["./cli/template/**/*.{ts,tsx}"],
+      extends: ["plugin:@typescript-eslint/disable-type-checked"],
+    },
+  ],
   rules: {
+    "@typescript-eslint/no-explicit-any": "error",
```

Comment:
We right now have `no-explicit-any` moved into the _`strict`_ config for v6. But I understand you were expecting it to be enabled. I'm now wondering if we should keep it at _`recommended`_?

For context, _`strict`_ is something we recommend for more confident TypeScript users. https://v6--typescript-eslint.netlify.app/linting/configs#strict

Reply:
so what's the main difference in strictness between

`strict-type-checked` and the old `recommended-requiring-type-checking`?

Reply:
The _`strict`_ & _`strict-type-checked`_ rulesets are a lot more opinionated. We phrase them as _"Additional strict rules that can also catch bugs but are more opinionated than recommended rules."_. In v6 they're supersets of the respective _`recommended`_ & _`recommended-type-checked`_ configs.

https://v6--typescript-eslint.netlify.app/rules/?supported-rules=xrecommended-strict shows the rules that are in _`strict*`_ and not _`recommended*`_.

Reply:
after reading through the list, i would definitely want about 2/3 of the strict rules and am neutral-but-could-be-pushed-positive on the remaining 1/3. 

what is the motivation for moving `no-explicit-any` to the strict rules?

Reply:
IIRC it was that a lot of projects end up needing to use `any` (or at least their developers think they do), and we're trying to not be in people's way as much with the recommended rulesets. In a sense this is us capitulating to how new & intermediate TypeScript developers write code (the occasional `any`, not always doing type safe things). Capitulating means less friction in turning on the recommended rulesets, at the cost of being less useful out-of-the-box.

Reply:
Do you think educating those users to try to use `unknown` instead could help adoption?

Reply:
Yes, definitely. A lot of these lint rules are better from user education - and can themselves act as user education!

Which reminds me, https://typescript-eslint.io/rules/no-explicit-any could do a bit better in that regard. Filed https://github.com/typescript-eslint/typescript-eslint/issues/7136. Thanks! 😄

Reply:
Just following up, I finally got around to documenting this: https://github.com/typescript-eslint/typescript-eslint/pull/8370